server/views/pages/bookedtoday/summary.njk

```diff
@@ -0,0 +1,76 @@
+{% extends "../../partials/layout.njk" %}
+{% from "components/summary-card.njk" import summaryCard %}
+{% from  "../../partials/expectedArrivalCardMacro.njk" import expectedArrivalCard, transferCard %}
+{% set pageTitle = "Person booked to arrive" %}
+
+{% block content %}
+    <div class="govuk-!-margin-bottom-8">
+
+        <div class="govuk-body">
+            {% if arrival.prisonNumber or arrival.pncNumber %}
+                <span class="govuk-caption-l">
+                    {% if arrival.prisonNumber %}Prison number: {{ arrival.prisonNumber }}{% endif %}{% if arrival.prisonNumber and arrival.pncNumber %} | {% endif %}{% if arrival.pncNumber %}PNC: {{ arrival.pncNumber }}{% endif %}
+                </span>
+            {% endif %}
+            <h1 class="govuk-!-margin-0">{{ arrival.lastName }}, {{ arrival.firstName }}</h1>
+
+            <div class="govuk-inset-text">
+                This summary contains information from the escort move and prison record.
+            </div>
+
+            <div class='govuk-grid-row'>
+                <div class='govuk-grid-column-two-thirds'>
+                    {{ summaryCard({
+                        classes: "govuk-!-margin-bottom-6",
+                        titleText: "Booking overview",
+                        rows: [
+                            {
+                                key: {  text: "Arrival Status" },
+                                value: { html: "<strong class=\"govuk-tag govuk-tag--grey\">" + arrival.potentialMatches[0].arrivalType + "</strong>" }
```

Comment:
This will always be "NOT ARRIVED" for pre arrival.

Reply:
Awesome, saves me having to map the types to strings! 😂